7_deploy_osh/roles/suse-dev-patcher/defaults/main.yml

```diff
@@ -1,46 +0,0 @@
----
-# defaults file for suse-dev-patcher
-
-devenv_root: /opt
-
-suse_dev_patcher_packages:
-  - autoconf
-  - git
-  - jq
-  - curl
-
-osh_repos:
-  openstack-helm-infra:
-    src: https://git.openstack.org/openstack/openstack-helm-infra
```

Comment:
why we are deleting frozen logic. I thought we will need that even after development cycle as we don't have concept of release branches in these projects. So we will need to freeze code to some point for product delivery

Reply:
The frozen logic is not deleted. It's merely moved to a common variable file, so that everything that requires freezing is in one place (vs defaults of different roles that all need updating). The manifest of what's frozen is moved to vars/manifest.yml.